backend/python/app/models/user_all.py

```diff
@@ -8,7 +8,7 @@
 
 
 class UserAll(db.Model):
-    __tablename__ = "users"
+    __tablename__ = "users_all"
```

Comment:
woah what 😳 im surprised this didnt cause any errors earlier. how did you discover this?

Reply:
me too, found this from doing the story test architecture exercise and looking through the models
i wonder if this is what caused the pytest issues we had? (though i think switching over to using migrations was a good idea since it is a closer imitation of what we do in normal development anyway)